src/content/en/2019/third-parties.md

```diff
@@ -120,9 +123,7 @@ The category breakdowns among script execution largely follow that of resource c
 
 While much could be said about every individual provider’s popularity and performance impact, this more opinionated analysis is left as an exercise for the reader and other purpose-built tools such as the previously mentioned [third-party-web](https://thirdpartyweb.today).
 
-## Analysis
-
-### Usage Patterns
+## Usage Patterns
```

Comment:
The heading structure looked ok to me. Each of these headings fit under "Analysis":

Usage Patterns
Generate and Consume Data
Monetize Web Traffic
Simplify Development

Reply:
I'm reading it as those three types (Generate..., Monetize and Simplify Development) are the Usage patterns. Hence they were sub headings for that heading. At which point it didn't seem to make sense to have Analysis as a heading, Usage as a sub-heading and the three under that as sub-sub-headings. Re-reading I still think this is the case so do think we should make this change.

Reply:
@patrickhulce can you clarify your intent?

Reply:
The `Usage Patterns` section was intended to be like an overview/intro for the `Analysis` section.

It'd make sense to me for any of the following to happen...

- Rename "Usage Patterns" heading to "Overview".
- Eliminate "Usage Patterns" heading.
- Rename "Analysis" to "Usage Patterns" and eliminate existing "Usage Patterns" heading (this change)

I'd have a slight preference for 2 to keep the top level headings fairly parallel but am fine with whatever y'all decide.

Reply:
I have a slight preference for 3 as think "Analysis" is a bit generic and "Usage Preferences" is more meaningful section name. Isn't the whole chapter (in fact the whole Almanac) "Analysis" after all? 

Will leave as is and resolve this, but feel free to re-open and add further suggestions. Or I could be persuaded to do 2 if you really feel strongly about it.

Reply:
the analysis generalism is a fair point. 3 is fine with me! :)